README.md

```diff
@@ -1,2 +1,31 @@
 # Hello-world
-Creating repo to demonstrate professional way of documenting project
+
+[Hello-world] hello Microverse Milestone completed
+
+-------------------------------------------------
+Built With
+-------------------------------------------------
+* HTML
+* CSS 
+
+-----------------------------------------------
+Key Features
+------------------------------------------------
+* index.html file
+* styles.css file
+* added linters 
+
+
+
+ 
+----------------------------------------------
+Author
+-----------------------------------------------
+Umarah Gul
```

Comment:
- [ ] kindly as requested by the previous reviewer consider making sure that every section in the provided template is present in your readme to make it more descriptive and also its mandatory to use the microverse Readme file template according to the project requirement Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] please follow this step in other not to make a mistakes kindly follow this step by step 
the only optional section is FAQ.
Students should delete the live demo section if the student decides not to deploy the application. If project requirements say students should deploy, they should put the link.
Make sure the student has added the section Future Features, including 1-3 potential features to be added.
Kindly remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
you can remove template logo. Optionally, ypou can replace with your own logo if you have one

Reply:
- [ ] [Required] Please as requested by the previous reviewer, kindly use this Microverse readme [template](https://github.com/microverseinc/readme-template). Please do not remove major sections in the template, you just need to update it to suit your project, the only main section that is optional is the FAQ section. :smile: 

- You can click on this [link](https://github.com/Alusp/World-Health-Assembly#world-health-assembly) to see a well-edited readme template.

Reply:
I have updated the feature branch but can you please guide me how to "REOPEN" a closed pull request? I dont seem to find my way around . Here the link https://github.com/umarahgul/Hello-world/pulls?q=is%3Apr+is%3Aclosed. I have tried many ways . @Alusp

Reply:
- [ ] @umarahgul Please create a new pull request and mention this pull request link in the new pull request summary then the new reviewer will be able to click on the link and be redirected to that pull request to check the previous required changes. To add a link in a PR Summary you will just write `Previous Pull Request: #1` I hope this will help you. :slightly_smiling_face:

Reply:
@Alusp U mean i shall mention the url of the older closed pull request in the summary of my  new pull request. Having an extra code review makes me look bad? what if I am again asked for changes ?